### ISSUE TITLE
Release 0.0.2

### ISSUE THREAD

**Issue by brandjon** [bazelbuild/rules_python#261]
Targeting Bazel 2.0 (or sooner).

This means we want compatibility with any incompatible changes in 2.0. Then we'll aim to be included in the Bazel Federation release that'll come out shortly after Bazel 2.0.

**Comment by pcn** [comment]
Where is this currently? I'd like to try using python3 with bazel, but following the "Getting Started" section, I end up with a WORKSPACE that looks like this:

```python
load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")

http_archive(
    name = "bazel_federation",
    url = "https://github.com/bazelbuild/bazel-federation/releases/download/0.0.1/bazel_federation-0.0.1.tar.gz",
    sha256 = "506dfbfd74ade486ac077113f48d16835fdf6e343e1d4741552b450cfc2efb53",
)

load("@bazel_federation//:repositories.bzl", "rules_python_deps")

rules_python_deps()

load("@bazel_federation//setup:rules_python.bzl",  "rules_python_setup")
rules_python_setup(use_pip=True)

load("@rules_python//python:pip.bzl", "pip3_import")

# Create a central repo that knows about the dependencies needed for
# requirements.txt.
pip3_import(   # or pip3_import
   name = "my_deps",
   requirements = ":requirements.txt",
)

# Load the central repo's install function from its `//:requirements.bzl` file,
# and call it.
load("@my_deps//:requirements.bzl", "pip_install")
pip_install()
``` 

and a BUILD that looks like this:
```
py_binary(
  name = "main",
  srcs = ["main.py"],
)
```

and a requirements.txt that just has something standard like `boto3` for e.g. results in this:

```
spacey@bagpig:~/dvcs/pcn/test-rules-python$ bazel build :main
ERROR: Failed to load Starlark extension '@rules_python//python:pip.bzl'.
Cycle in the workspace file detected. This indicates that a repository is used prior to being defined.
The following chain of repository dependencies lead to the missing definition.
 - @rules_python
This could either mean you have to add the '@rules_python' repository with a statement like `http_archive` in your WORKSPACE file (note that transitive dependencies are not added automatically), or move an existing definition earlier in your WORKSPACE file.
ERROR: cycles detected during target parsing
INFO: Elapsed time: 0.063s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
```
So, I see that I'm using bazel 2.0, so this seems like the relevent GH issue, but I'm not sure what to ask for in this, beyond "how is this supposed to work, and can save that somehow?

**Comment by gertvdijk** [comment]
I'd like to see a new release too. In the meantime, use the master branch in order to use this with current stable versions of Bazel and Python 3.

**Comment by dayfine** [comment]
ping. There should be periodically releases that people can use with up-to-date README.md that matches the latest release.

Or at least README should reflect that the release process is halt and `master` should be used for latest changes.

**Comment by thundergolfer** [comment]
Related: https://github.com/bazelbuild/rules_python/issues/292

**Comment by brandjon** [comment]
Closing as @thundergolfer has released 0.0.2.
